scripts/systemd/nut-driver-enumerator.service.in

```diff
@@ -21,6 +21,7 @@ Type=oneshot
 # nut-server which is the primary concerned dependency at the moment, so we
 # don't want it to fail the unit (when it can't restart).
 Environment=REPORT_RESTART_42=no
+EnvironmentFile=-@CONFPATH@/nut.conf
```

Comment:
Will the `... ; export ALLOW_NO_DEVICE` shell syntax work with systemd's `EnvironmentFile` parser? Have not tested myself.

https://www.freedesktop.org/software/systemd/man/systemd.exec.html

Reply:
Very good catch, thanks. By coincidence, we check the first 4/5 chars of the envvar so find `true`/`false` anyway, so this slipped past the tests. But yes, systemd puts everything after `=` into the envvar value:

````
root@debian8:~# cat /run/systemd/system/envtest.service 
[Service]
EnvironmentFile=/run/envfile
ExecStart=/bin/sh -c 'set | grep TEST'
Type=oneshot
````

````
root@debian8:~# cat /run/envfile
TEST=123; export TEST
TESTML=456
export TESTML
````

Result of experiment: TEST is weird and wrong, TESTML is as expected:
````
# systemctl status envtest
Oct 12 11:07:59 debian8 systemd[1]: Starting envtest.service...
Oct 12 11:07:59 debian8 sh[26737]: TEST='123; export TEST'
Oct 12 11:07:59 debian8 systemd[1]: Started envtest.service.
Oct 12 11:08:40 debian8 systemd[1]: Starting envtest.service...
Oct 12 11:08:40 debian8 sh[26771]: TEST='123; export TEST'
Oct 12 11:08:40 debian8 sh[26771]: TESTML='456'
Oct 12 11:08:40 debian8 systemd[1]: Started envtest.service.
````

Reply:
Split the example into two lines, and added a general comment about expected syntax for the file.